ValgrindCI/render.py

```diff
@@ -143,16 +149,20 @@ def _extract_data_per_source_file(
         else:
             filename = source_file
 
-        with open(filename, "r") as f:
-            for l, line in enumerate(f.readlines()):
-                klass = None
-                issue = None
-                if l + 1 in error_lines:
-                    klass = "error"
-                    issue = self._extract_error_data(
-                        src_data, l + 1, lines_before, lines_after
+        filename = self._data.substitute_path(filename)
+        try:
+            with open(filename, "r", errors="replace") as f:
```

Comment:
There again, what is the point of using `errors="replace"` when reading a file ?